polkadot/runtime/parachains/Cargo.toml

```diff
@@ -19,6 +19,8 @@ serde = { features = ["alloc", "derive"], workspace = true }
 derive_more = { workspace = true, default-features = true }
 bitflags = { workspace = true }
 
+sc-network = { workspace = true }
```

Comment:
This looks like a really heavy dependency to bring into the runtime just for this constant, not sure if the compiler is smart enough to saves us all the troubles we might have.

Reply:
Any better ideas? Since #5753 we have a dedicated constant for PoV response size, but it is also situated in a location we wouldn't want to import. Maybe we should move that constant somewhere but I'm struggling to find a good place for it. `polkadot-parachain-primitives` is perfect, as it relates to parachains and is shared between the node and the runtime, but it seems to me it's strange to have a network-related constant there.

Reply:
Also, this is low level networking, doesn't make sense for the runtime to depend on it.

Reply:
> Any better ideas? 

an alternative is in comment https://github.com/paritytech/polkadot-sdk/pull/5887#discussion_r1782594505